### PR TITLE
Split distortion into concealment and acceleration/deacceleration

### PR DESCRIPTION
**Pull request by pnts-se-whereby**
<details>
  <summary>📦 Published PR as canary version: <code>1.7.7--canary.68.7869334338.0</code></summary>
  <br />
  
  :sparkles: Test out this PR locally via:
  
  ```bash
  npm install @whereby/jslib-media@1.7.7--canary.68.7869334338.0
  # or 
  yarn add @whereby/jslib-media@1.7.7--canary.68.7869334338.0
  ```
</details>

### CODE REVIEW

File: src/webrtc/stats/IssueMonitor/index.js [hunk 1]
```diff
@@ -112,13 +112,22 @@ const issueDetectors = [
...
         enabled: ({ hasLiveTrack, ssrc0, kind }) => hasLiveTrack && ssrc0 && kind === "audio",
         check: ({ ssrc0 }) =>
-            ssrc0.bitrate &&
-            ssrc0.direction === "in" &&
-            ssrc0.audioLevel >= 0.001 &&
-            Math.max(ssrc0.audioConcealment, ssrc0.audioAcceleration, ssrc0.audioDeceleration) >= 0.1,
+            ssrc0.bitrate && ssrc0.direction === "in" && ssrc0.audioLevel >= 0.001 && ssrc0.audioConcealment >= 0.1,
+    },
+    {
+        id: "deaccelerated",
```
Comment (geirbakke) [inline src/webrtc/stats/IssueMonitor/index.js:121]: decelerated

File: src/webrtc/stats/IssueMonitor/index.js [hunk 2]
```diff
@@ -205,15 +214,37 @@ const metrics = [
...
+            hasLiveTrack &&
+            ssrc0 &&
+            ssrc0.bitrate &&
+            ssrc0.direction === "in" &&
+            kind === "audio" &&
+            ssrc0.audioLevel >= 0.001,
+        value: ({ ssrc0 }) => ssrc0.audioConcealment,
+    },
+    {
+        id: "deacceleration",
```
Comment (geirbakke) [inline src/webrtc/stats/IssueMonitor/index.js:228]: deceleration

**Comment by geirbakke** [comment]
ah, you fixed as I reviewed

File: src/webrtc/stats/IssueMonitor/index.js (hunk 2 above)
Comment (pnts-se-whereby) [reply src/webrtc/stats/IssueMonitor/index.js]: fixed in [180fdb9](https://github.com/whereby/jslib-media/pull/68/commits/180fdb92619036087f420a1083bf3396c379749f)

File: src/webrtc/stats/IssueMonitor/index.js (hunk 1 above)
Comment (pnts-se-whereby) [reply src/webrtc/stats/IssueMonitor/index.js]: fixed in [180fdb9](https://github.com/whereby/jslib-media/pull/68/commits/180fdb92619036087f420a1083bf3396c379749f)

**Pull request merged**